container/common/helpers.go

```diff
@@ -109,8 +109,9 @@ func GetSpec(cgroupPaths map[string]string, machineInfoFactory info.MachineInfoF
 				val, err := strconv.ParseUint(quota, 10, 64)
 				if err != nil {
 					klog.Errorf("GetSpec: Failed to parse CPUQuota from %q: %s", path.Join(cpuRoot, "cpu.cfs_quota_us"), err)
+				} else {
+					spec.Cpu.Quota = val
 				}
```

Comment:
consider using `readUnit64` function to read quota, if I see correctly this function makes what you made here